City-of-Pori/pori-kada
# PORI-X Fixed duplicate top menu on visitpori news archive

To test:

1. drush fra -y && drush cc all
2. Go to local.visitpori.fi/news-archive and make sure there is no duplicate top menu.
